docs/PhDInterns.md

```diff
@@ -0,0 +1,29 @@
+---
+hide:
+  - navigation
+---
+
+# PhD Intern Scheme
```

Comment:
should it not be either "PhD Internship Scheme" or "PhD Interns Scheme"?

Reply:
Will correct to be consistent to "PhD Internship Scheme"